crates/fuzzing/src/generators/memory.rs

```diff
@@ -5,6 +5,75 @@ use arbitrary::{Arbitrary, Unstructured};
 use std::ops::Range;
 use wasmtime::{LinearMemory, MemoryCreator, MemoryType};
 
+/// A description of a memory config, image, etc... that can be used to test
+/// memory accesses.
+#[derive(Debug)]
+#[allow(missing_docs)]
+pub struct MemoryAccesses {
+    pub memory_config: NormalMemoryConfig,
+    pub image: HeapImage,
+    pub offset: u32,
+    pub growth: u32,
+}
```

Comment:
Mind adding a memory64 knob to this as well? Also mind adding some docs for what each field member is? For example at a glance I'm not sure what `growth` is.

Reply:
Also could this grow a fully general `WasmtimeConfig`? It would be good to test various compilation settings, Winch, the pooling allocator, etc.

Reply:
Good idea, will do.